zio-http/src/main/scala/zio/http/ZClientAspect.scala

```diff
@@ -165,6 +166,50 @@ object ZClientAspect {
       }
     }
 
+  /**
+   * Client aspect that updates the proxy settings of specified client
+   */
+  final def proxyClient(proxy: Proxy) (implicit  trace: Trace): ZClientAspect[Nothing, Any, Nothing, Body, Nothing, Any, Nothing, Response] =
```

Comment:
I think at this point you can delete the aspect and just make `proxy` an operator on `ZClient` like `ssl`.